osu.Game/Screens/Select/BeatmapDetailAreaTabControl.cs

```diff
@@ -69,6 +74,13 @@ private void load(OsuColour colour, OsuConfigManager config)
             tabs.Current.TriggerChange();
         }
 
+        protected override void Update()
+        {
+            base.Update();
+
+            tabsContainer.Padding = new MarginPadding { Right = modsCheckbox.Width };
```

Comment:
Just to clarify, is this done in `Update` because the checkbox changes width dynamically?

Reply:
Checking again it does change width when the the text gets bold when checked, so this is probably right.

Initially, I just didn't want to specify some random fixed number as the padding.